### PR TITLE
feat(doc): Document che-editors.yaml and che-plugins.yaml files

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Document che-editors.yaml and che-plugins.yaml files

### What issues does this PR fix or reference?
https://github.com/eclipse/che-plugin-registry/pull/777

### How to test this PR?
Browse rendered MD file at https://github.com/benoitf/che-plugin-registry/blob/contributing/CONTRIBUTE.md

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

Change-Id: Ide12da7213f5219df0d891e9b7543e1341c6d6d8
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -68,6 +68,216 @@ Here are all the supported values, including optional ones:
...
+        # (OPTIONAL) mount the source code of the projects in the container
+        # Should be true for the editor's container as usually editor is to open files from projects
+        mountSources: true
+        # (OPTIONAL) ports exposed by this container
+        ports:
+         - 
+           # Should at least expose the port used by the default endpoint
+           exposedPort: 8080
+        memoryLimit: "512M"          
+          # Directory where the Dockerfile that builds this extension is located
```
Comment (svor) [inline CONTRIBUTE.md:146]: i think the attribute is missing or need to remove this comment

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -68,6 +68,216 @@ Here are all the supported values, including optional ones:
...
+        # (OPTIONAL) The memory limit of the container
+        memoryLimit: "1500Mi"
```
Comment (svor) [inline CONTRIBUTE.md:148]: two times memoryLimit

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -68,6 +68,216 @@ Here are all the supported values, including optional ones:
...
+        mountSources: true
+        # (OPTIONAL) ports exposed by this container
+        ports:
+         - 
+           # Should at least expose the port used by endpoint (if any)
+           exposedPort: 8080
+        memoryLimit: "512M"          
+          # Directory where the Dockerfile that builds this extension is located
+        # (OPTIONAL) The memory limit of the container
+        memoryLimit: "1500Mi"
```
Comment (svor) [inline CONTRIBUTE.md:252]: two times memoryLimit

File: CONTRIBUTE.md [hunk 4]
```diff
@@ -68,6 +68,216 @@ Here are all the supported values, including optional ones:
...
+          path: "/home/theia/.m2"
+        # (OPTIONAL) mount the source code of the projects in the container
+        mountSources: true
+        # (OPTIONAL) ports exposed by this container
+        ports:
+         - 
+           # Should at least expose the port used by endpoint (if any)
+           exposedPort: 8080
+        memoryLimit: "512M"          
+          # Directory where the Dockerfile that builds this extension is located
```
Comment (svor) [inline CONTRIBUTE.md:250]: do we need this?

**Comment by benoitf** [comment]
@svor thanks for the review, I've updated the PR

**Review by svor — APPROVED** [review]
LGTM

**Pull request merged**